images/base/Dockerfile

```diff
@@ -69,18 +68,27 @@ RUN clean-install \
     && rm -f /lib/systemd/system/sockets.target.wants/*initctl* \
     && rm -f /lib/systemd/system/basic.target.wants/* \
     && echo "ReadKMsg=no" >> /etc/systemd/journald.conf \
-    && systemctl enable containerd \
-    && export ARCH=$(dpkg --print-architecture | sed 's/ppc64el/ppc64le/' | sed 's/armhf/arm/') \
-    && curl -fSL -o /usr/local/bin/ctr \
-      "https://storage.googleapis.com/bentheelder-kind-dev/containerd/linux/${ARCH}/ctr" \
-    && chmod +x /usr/local/bin/ctr \
     && echo "done installing packages"
 
 # override the rp_filter settings to enable calico cni to "just work"
 COPY 10-network-security.conf /etc/sysctl.d/
 
-# add overrides to containerd
-COPY 10-limits.conf 10-restart.conf /etc/systemd/system/containerd.service.d/
+# Install containerd nigthly binaries so we can use latest features not present in Ubuntu
+# https://github.com/aojea/containerd-nightly-releases/releases/download/v0.0.0-20191003172204-be6bead4/containerd-0.0.0-20191003172204-be6bead4.linux-amd64.tar.gz
+ARG CONTAINERD_VERSION="0.0.0-20191003172204-be6bead4"
+ARG CONTAINERD_BASE_URL="https://github.com/aojea/containerd-nightly-releases/releases/download/"
+RUN export ARCH=$(dpkg --print-architecture | sed 's/ppc64el/ppc64le/' | sed 's/armhf/arm/') \
+    && export CONTAINERD_TARBALL="v${CONTAINERD_VERSION}/containerd-${CONTAINERD_VERSION}.linux-${ARCH}.tar.gz" \
+    && echo "${CONTAINERD_URL}" \
+    && export CONTAINERD_URL="${CONTAINERD_BASE_URL}${CONTAINERD_TARBALL}" \
+    && curl -sSL --retry 5 --output /tmp/containerd.tgz "${CONTAINERD_URL}" \
+    && sha256sum /tmp/containerd.tgz \
```

Comment:
It’s not necessary？ We don’t check it

Reply:
copy and paste leftover from the cni one